### ISSUE TITLE
A11y_AzureDataStudio_JuypyterBook_ScreenReader: After clicking enter on any saved books, make the changes in right panel but NVDA doesn’t announce the status message.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7795]
[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureDataStudio;#A11yWCAG2.1;#A11yTCS;#WCAG4.1.3;#DesktopApp;#JAWS;#NVDA;#SQL Azure Data Studio;#Win10;#A11yExclusion

**Environment Details:	**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA,JAWS

**Steps to Reproduce:**
1.Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the "jupyter books" icon.
4.Press CRTL+SHIFT+P and search "jupyter books: SQL Server 2019 Guide" 
5.Navigate to the left section of jupyter books under the untitled books and click on "save" icon .
6.Verify that all the elements of "Save juypyter book" are accessible and MAS Compliant.
7.Start NVDA.	
8.Navigate till tree item using arrows and select that item.
9.Observe the announcement by screen reader. 
 
**Actual:**
Screen reader not announcing the status message as after clicking on any of tree item right panel screen gets changed.
For example, after clicking on “TSG021 - Get Kubernetes cluster information”, the screen reader doesn’t announce anything. 
	
**Expected:** 
Screen reader should announce the status message .
For example, after clicking on “TSG021 - Get Kubernetes cluster information”, the screen reader should announce that  “TSG021 - Get Kubernetes cluster information screen opened " . 	

**User Impact:**
Screen reader user will it difficult to navigate the screens because after clicking on any control new screens open but screen reader doesn’t make any announcement regarding that .

**Attachment for Reference:**
[7795_A11y_AzureDataStudio_JuypyterBook_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3742602/7795_A11y_AzureDataStudio_JuypyterBook_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by MaddyDev** [comment]
https://github.com/microsoft/vscode/issues/88800 on vscode

**Comment by hannahqin** [comment]
Marking this for Exclusion. As stated above, this issue is upstream with VS Code and is being tracked here: https://github.com/microsoft/vscode/issues/88800. 

**Quality of Experience:** manageable
Experience may be mildly annoying upon discovery of this feature since the user may not know that each tree item opens new content on the main screen. However, with simple training or knowledge of this bug from the Conformance Document, users can easily overcome this issue.

**Importance of Experience:** somewhat important
Jupyter Books can be helpful to certain customers as a tool for collaboration or documentation, but they are currently infrequently used and not essential to the core use cases the product.

This bug has been documented in our **AAT Conformance Doc:** https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde?tab=default

@12-shweta, please work with a SME to review.

**Comment by 12-shweta** [comment]
As discussed with SME, closing this bug as the non-blocking issue.
